docs/non-normative/db-sql-test-cases.json

```diff
@@ -0,0 +1,219 @@
+        ],
+        "db.query.summary": "SELECT"
+    },
+    {
+        "name": "in_clause",
+        "db.system.name": "other_sql",
+        "query": "SELECT * FROM table WHERE value IN (123, 456, 'abc')",
+        "db.query.text": [
+            "SELECT * FROM table WHERE value IN (?)",
+            "SELECT * FROM table WHERE value IN (?, ?, ?)"
+        ],
+        "db.query.summary": "SELECT table"
+    },
+    {
+        "name": "comments",
+        "db.system.name": "other_sql",
+        "query": "SELECT column -- end of line comment\nFROM /* block \n comment */ table",
+        "db.query.text": [
+            "SELECT column \nFROM  table"
+        ],
+        "db.query.summary": "SELECT table"
+    },
+    {
+        "name": "insert_into_select",
+        "db.system.name": "other_sql",
+        "query": "INSERT INTO shipping_details\n(order_id,\naddress)\nSELECT order_id,\naddress\nFROM   orders\nWHERE  order_id = 1",
+        "db.query.text": [
+            "INSERT INTO shipping_details\n(order_id,\naddress)\nSELECT order_id,\naddress\nFROM   orders\nWHERE  order_id = ?"
+        ],
+        "db.query.summary": "INSERT shipping_details SELECT orders"
+    },
+    {
+        "name": "select_nested_query",
+        "db.system.name": "other_sql",
+        "query": "SELECT order_date\nFROM   (SELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)",
+        "db.query.text": [
+            "SELECT order_date\nFROM   (SELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)"
+        ],
+        "db.query.summary": "SELECT SELECT orders customers"
+    },
+    {
+        "name": "select_nested_query_case_preserved",
+        "db.system.name": "other_sql",
+        "query": "SELEcT order_date\nFROM   (sELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)",
+        "db.query.text": [
+            "SELEcT order_date\nFROM   (sELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)"
+        ],
+        "db.query.summary": "SELEcT sELECT orders customers"
+    },
+    {
+        "name": "case_preserved",
+        "db.system.name": "other_sql",
+        "query": "SELEcT order_date\nFROM ORders",
+        "db.query.text": [
+            "SELEcT order_date\nFROM ORders"
+        ],
+        "db.query.summary": "SELEcT ORders"
+    },
+    {
+        "name": "cross_join",
+        "db.system.name": "other_sql",
+        "query": "SELECT * FROM Orders o CROSS JOIN OrderDetails od",
+        "db.query.text": [
+            "SELECT * FROM Orders o CROSS JOIN OrderDetails od"
+        ],
+        "db.query.summary": "SELECT Orders OrderDetails"
+    },
+    {
+        "name": "cross_join_comma_separated_syntax",
+        "db.system.name": "other_sql",
+        "query": "SELECT * FROM Orders o, OrderDetails od",
+        "db.query.text": [
+            "SELECT * FROM Orders o, OrderDetails od"
+        ],
+        "db.query.summary": "SELECT Orders"
```

Comment:
why not `SELECT Orders, OrderDetails` ?

Reply:
This was a mistake. Fixed. It should be `SELECT Orders OrderDetails` (i.e., no comma between Orders and OrderDetails)